airbyte-api/src/main/openapi/config.yaml

```diff
@@ -3987,8 +4023,100 @@ components:
           $ref: "#/components/schemas/ConnectionId"
         state:
```

Comment:
switch this to legacy?

Reply:
I am wondering about compatibility issues. I have no idea about how this endpoint is used (apart from it not being use in the UI). I some users have a script fetching the state, it should remains the same.